frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -523,6 +523,16 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                                           : undefined,
                               }
                             : null,
+
+                        featureFlags[FEATURE_FLAGS.REVENUE_ANALYTICS]
```

Comment:
going above replay in the menu... that's a "Request changes" right there 🤣

Reply:
Yeah, I just did this because B2B Analytics is at the top, but I believe we should be ranking these by revenue (or something like this) instead, and in that case, replay goes 📈 

I won't care much about this because team-devex will change this anyway, and will instead hide these, and this will be moot

Reply:
nah, i was only joking... i think it makes sense for blah analytics to be together

ultimately i think we should let people choose what goes on the home page... 

then the real fight begins!